Stylo: Implement -moz-text-align-last

Please go through [the guide](https://github.com/servo/servo/wiki/Stylo-hacking-guide) if you wish to hack on stylo.

[`text-align-last`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align-last) is a keyword property ([implementation guide](https://github.com/servo/servo/wiki/Stylo-hacking-guide#implementing-a-simple-keyword-property)) with the set of possible values being `auto | start | end | left | right | center | justify`. `auto` is the default.

In Firefox, this property is called `-moz-text-align-last` (so we will implement it with that name), but behaves the same way. The constant prefix is [`NS_STYLE_TEXT_ALIGN`](https://dxr.mozilla.org/mozilla-central/rev/4b9944879c9a60a9aba4a744a7401bc38e0f39c4/layout/style/nsStyleConsts.h#863)

File: https://github.com/servo/servo/blob/master/components/style/properties/longhand/inherited_text.mako.rs

Testcase: https://gist.github.com/Manishearth/daaecff21aecb46d5d0b50bbfcd9c130 . It should look the same in Stylo as it does in vanilla Firefox after the changes have been made.

I ran `./mach clobber && ./mach bootstrap && ./mach build -j3` and got this:
```
0:08.76 Traceback (most recent call last):
 0:08.76   File "/Users/awesomeaniruddh/rust/stylo-flat/configure.py", line 107, in <module>
 0:08.76     sys.exit(main(sys.argv))
 0:08.76   File "/Users/awesomeaniruddh/rust/stylo-flat/configure.py", line 26, in main
 0:08.76     sandbox.run(os.path.join(os.path.dirname(__file__), 'moz.configure'))
 0:08.76   File "/Users/awesomeaniruddh/rust/stylo-flat/python/mozbuild/mozbuild/configure/__init__.py", line 256, in run
 0:08.76     raise InvalidOptionError('Unknown option: %s' % without_value)
 0:08.76 mozbuild.configure.options.InvalidOptionError: Unknown option: --enable-warnings-as-errors
 0:08.81 *** Fix above errors and then restart with               "/Applications/Xcode.app/Contents/Developer/usr/bin/make -f client.mk build"
 0:08.81 make: *** [configure] Error 1
```

I ran the build for 48 minutes and it failed with this message:
```
48:02.42 libdombindings_test_s.a.desc
48:02.60 make[3]: *** [compile] Error 2
48:02.60 make[2]: *** [default] Error 2
48:02.62 make[1]: *** [realbuild] Error 2
48:02.62 make: *** [build] Error 2
48:02.63 403 compiler warnings present.
48:02.85 /usr/local/bin/terminal-notifier -title Mozilla Build System -group mozbuild -message Build failed
```

Gonna run it with your mozconfig now.

--- a/components/style/properties/longhand/inherited_text.mako.rs
+++ b/components/style/properties/longhand/inherited_text.mako.rs
@@ -395,6 +395,13 @@ ${helpers.single_keyword("text-justify",
                          products="servo",
                          animatable=False)}
 
+${helpers.single_keyword("-moz-text-align-last",
+                         "auto start end left right center justify",
+                         products="gecko",
+                         gecko_constant_prefix="NS_STYLE_TEXT_ALIGN",
+                         gecko_ffi_name="mTextAlignLast",
+                         animatable=False)}
+
 <%helpers:longhand name="-servo-text-decorations-in-effect"
                    derived_from="display text-decoration"
                    need_clone="True" products="servo"

